cmd/fetch_test.go

```diff
@@ -146,7 +153,7 @@ func TestFetchCommand(t *testing.T) {
 	}
 
 	// Regex that replaces the timestamp with just static text
-	fixTs := regexp.MustCompile(`"timestamp":"[0-9TZ.:-]*"`)
+	fixTs := regexp.MustCompile(`"timestamp":"[0-9TZ.:+-]*"`)
```

Comment:
@dustmop just wanted to higlight that I've added the `+` here as the test was failing in my case having a `+01:00` at the end probably due to my TZ. Also need to update the tests to use this bellow as well.